### PR TITLE
Table: enables SSR rendering and compatibility

### PR DESCRIPTION
**Pull request by boyangwang**
This commit enables rendering of vnode and DOM during SSR
Changes are:
```
	modified:   examples/play/index.vue
	modified:   packages/table/src/table-body.js
	modified:   packages/table/src/table-column.js
	modified:   packages/table/src/table-store.js
	modified:   packages/table/src/table.vue
```

`packages/table/src/table-body.js`
give colspan rowspan default value of 1

`packages/table/src/table-column.js`
add insertColumn action to created hook

`packages/table/src/table-store.js`
updateColumns() is a safe operation, do it even if $ready===false

`packages/table/src/table.vue`
do updateColumns() in created hook

`examples/play/index.vue`
contains an example. Not intended to commit, for testing & verification purpose

Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md) | [Español](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.es.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by eeeeeeeason** [comment]
我该怎么拿到你这次提交。能正常获取store里的信息到window.__INITIAL_STATE__，我特喵的el-table里的数据是空的。

File: examples/play/index.vue [hunk 1]
```diff
@@ -1,8 +1,88 @@
 <template>
   <div style="margin: 20px;">
+        <el-table
```
Comment (island205) [inline examples/play/index.vue]: don't commit these testing code.

**Comment by huangdandan** [comment]
您好，按照上述操作完成后，新的lib 中的table会报如下错误，vue.js:637 [Vue warn]: Unknown custom element: <el-table-column> - did you register the component correctly? For recursive components, make sure to provide the "name"   ##option.

**Comment by qiwang97** [comment]
> 您好，按照上述操作完成后，新的lib 中的table会报如下错误，vue.js:637 [Vue warn]: Unknown custom element: - did you register the component correctly? For recursive components, make sure to provide the "name" ##option.

https://www.cnblogs.com/ssjd/p/14171245.html

**Pull request closed without merging**